### ISSUE TITLE
Parts of the homepage hidden with resolutions <1200px wide

### ISSUE THREAD

**Issue by hhiru** [nyaadevs/nyaa#369]
When you resize the homepage of Nyaa the right part of the table is hidden cutting out part of date, seed/peer and total-downloads. This is an animated sample of the unresponsive behavior https://my.mixtape.moe/tufqtv.webm

![image](https://user-images.githubusercontent.com/11262717/30521448-06300c82-9bc0-11e7-9d8c-9acc6c9b097d.png)

**Comment by agungsijawir** [comment]
it is from Bootstrap. The layout automatically wraps or hide based on screen size.
if you scroll down to end page, you'll see horizontal scrollbar / try to swap to left on the torrent list.

and I think this is not a bug or something unless otherwise, you want to propose "The New" layout for nyaa :)

**Comment by msikma** [comment]
I think what @hhiru means is the table is partly occluded even though you could just reduce the size of the *name* column and have everything work fine. If the <1200px breakpoint were modified so that the `.torrent-list>tbody>tr>td` selector has a `max-width` of `435px`, everything fits for that size. That might be a nice simple change to do for people who are on that breakpoint.

**Comment by S-ed** [comment]
For some reason, sources have only have minified bootstrap files.

**Comment by msikma** [comment]
> For some reason, sources have only have minified bootstrap files.

The CSS file is right here: https://github.com/nyaadevs/nyaa/blob/master/nyaa/static/css/main.css

**Comment by S-ed** [comment]
@msikma Are you sure? It's a tiny spec of the style I'm editing by now.
I've disabled main.css in browser and it doesn't changed much. only minor fixes.

**Comment by ReimuHakurei** [comment]
Why would you expect a non minified Bootstrap CSS file?

**Comment by TheAMM** [comment]
@ReimuHakurei Idea is to have source files in a readable & editable format. We're packing Flask-Assets for minifying CSS (and compiling SASS or babeling Javascript, etc), but currently not using it.
Getting a vanilla bootstrap-min.css + nyaastrap.css and mashing them together/linking them separately would make CSS adjustments... better.

**Comment by ReimuHakurei** [comment]
We have changes to massive portions of bootstrap.css, made with the online
Bootstrap customizer.

If we had a stock bootstrap.css and overrides, the override file would be
massive.

All we need is the minified Bootstrap stuff, because we are not going to
make any direct changes. Any changes need to be made with the customizer
tool, as described in the HTML comment in the layout template.

**Comment by S-ed** [comment]
@ReimuHakurei fair enough. Now when I've opened the `bootstrap.min.css` from the repo I could see the comments. Strangely enough, they are missing from dark version. So no link to GIST `config.json`.
